fvm/src/machine/default.rs

```diff
@@ -88,16 +93,36 @@ where
             ));
         }
 
-        let bstore = BufferedBlockstore::new(blockstore);
-
-        let state_tree = StateTree::new_from_root(bstore, &context.initial_state_root)?;
+        // Load the built-in actors manifest.
+        let builtin_actors: Manifest = blockstore
+            .get_cbor(&builtin_actors)
```

Comment:
The idea we discussed was linking to these CIDs from the init actor state. But, we can't do this _before_ the migration because that would alter the init actor state and break consensus. So the solution I gravitate towards is:

1. Make the node (re-)import the CAR on the migration epoch.
2. Link the CodeCIDs in the init actor state.

In practical terms, we could do away with (2), because every actor will end up being linked from the state tree anyway:
1. Singleton actors are permanent.
2. Non-singleton actors cannot be destroyed, and there's many of each in the network already.